build-system/pr-check/dist-bundle-size.js

```diff
@@ -55,6 +69,7 @@ async function main() {
     timedExecOrDie('gulp dist --fortesting');
     timedExecOrDie('gulp bundle-size --on_push_build');
     uploadDistOutput(FILENAME);
+    buildAndUploadExperiments_();
```

Comment:
I'm concerned this will double the running time of the `build` stage of push builds, and potentially slow down the release process. Would it be possible to do this in a separate parallel job?

Reply:
Yeah good idea, here's what it could look like on push builds. 
![image](https://user-images.githubusercontent.com/44627152/62317070-dbdf0200-b466-11e9-9224-5c89d3350d18.png)

Reply:
The job duration is just for 1 experiment, though. I'll have to figure out how to run multiple in parallel.

Reply:
Nice! New matrix LGTM.

Yeah, fitting in more experiments is an interesting problem. Possible ideas:

- Build and test experiments only for canary and release branches, and for PRs that modify experimental code
- Optimize experimental builds to incrementally rebuild just the extensions that contain experimental code, and reuse the build output (not sure how feasible this is)

Reply:
So I figured out run test suites in parallel on 1 vm using multiple node child processes! It works fine with e2e tests after removing the hardcoded 8000 port in some places. Unfortunately it's a lot hairier for integration tests so I'm not running those yet.

For running builds in parallel, still haven't figured that out / whether it's feasible.